src/main/common/sdft.c

```diff
@@ -130,7 +130,7 @@ FAST_CODE void sdftWinSq(const sdft_t *sdft, float *output)
     float re;
     float im;
 
-    for (int i = (sdft->startBin + 1); i < sdft->endBin; i++) {
+    for (int i = sdft->startBin ; i <= sdft->endBin; i++) {
```

Comment:
This can introduce a critical **array index out of bounds** error and can cause the FC to lock up with unfortunate dyn notch settings.

Reply:
fixed